db/migrate/20240909180908_add_ai_summary_type_column.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+class AddAiSummaryTypeColumn < ActiveRecord::Migration[7.1]
+  def change
+    add_column :ai_summaries, :summary_type, :string
```

Comment:
I wonder if we should use an enum here :thinking:

Reply:
Much better, yes